content/References/ExtensionCalculator.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "RAIDZ Extension Calculator"
-description: "Calculator to determine usable ZFS capacity and other ZFS pool metrics, and to compare storage pool layouts."
-weight: 90
+description: "RAIDZ expansion adds disks to an existing RAIDZ VDEV. Use this calculator to estimate the storage capacity efficiency of an extended ZFS RAIDZ VDEV."
```

Comment:
This is the brief description that shows up as part of the children shortcode, so the firs sentence really doesn't fit the model/parallel construction aspect of all the other article descritpions.  If you really want to keep this information, maybe this is a better way. Thoughts?
```suggestion
description: "Use this calculator to estimate the storage capacity efficiency of an extended ZFS RAIDZ VDEV. RAIDZ expansion adds disks to an existing RAIDZ VDEV."
```